.github/workflows/dotnet-core-linux.yml

```diff
@@ -20,17 +20,17 @@ jobs:
     - name: Setup .NET Core 2.1
       uses: actions/setup-dotnet@v1
       with:
-        dotnet-version: 2.1.807
+        dotnet-version: 2.1.811
```

Comment:
I was getting an error that 2.1.807 could not be found. Bumped to latest.

Reply:
we could change to 2.1.x and 3.1.x and remove the rsync script.

Reply:
I ended up getting a conflict on the .net5 PR so while I was in there merging I switched to the ".x" for 2.1 & 3.1 versions.